docs/2.9.0/docs/canton/usermanual/performance.rst

```diff
@@ -506,74 +506,111 @@ If you submit a ledger API command (which can have multiple Daml commands), then
+      -- they don't get merged into a single view if this choice is called independently each time (multiple root actions)
       choice Foo : ()
-         controller owner 
-         do 
-         return ()
+         controller owner
+         do
+            return ()
 
    template BatchFoo
-   with 
-      owner : Party 
-   where 
-      signatory owner 
+   with
+      owner : Party
+   where
+      signatory owner
 
       choice Run : ()
-         with  
+         with
          batch : [ContractId Example]
          obs : Party
          -- The observer here is a choice observer. Therefore, Canton will
-         -- ship a single view with the top node of this choice if the observer of the 
-         -- choice aligns with the observer of the contract.       
+         -- ship a single view with the top node of this choice if the observer of the
+         -- choice aligns with the observer of the contract. In other words, either they are the same or the new observer
+         -- is hosted by any of the contract's informee participants.
          observer obs
-         controller owner 
+         controller owner
          do
-         forA_ batch (`exercise` Foo)
+            forA_ batch (`exercise` Foo)
 
-
-As a rule, the number of views should depend on the number of stakeholder groups you have in your batch choice, 
-not the number of "batches" you process in parallel within one command.
+As a rule, the number of views should depend on the number of informee participants
+you have in your batch choice that is not a subset of the parent's contract informee participants,
+not the number of "batches" you process in parallel within one command. A participant stakeholder group is formed
+by aggregating the participants that host each stakeholder, or, in other words, the set participants that
+need to see a particular view.
 
 The informees for the different type of transaction tree nodes are (also see :ref:`da-model-projections`):
-   * create: signatories, observers 
+   * create: signatories, observers
    * consuming exercise: signatories, observers, stakeholders of the choice (controller, choice observers)
    * nonconsuming exercise: signatories, stakeholders of the choice (controller, choice observers), but not the observers of the contract
    * fetch: signatories + actors of the fetch, which are all stakholders which are in the authorization context that the fetch executed in
    * lookupByKey: only key maintainers
 
+
+Strategy for Protocol Versions 5 and Lower
+++++++++++++++++++++++++++++++++++++++++++
+
+For protocol versions 5 and lower, Canton creates a view for every action node in the transaction tree if the informees
+or confirming parties change between the node and its parent. Therefore, in order to minimize the number of views,
+you should try to minimize the number of changes of informees and confirming parties.
+
+When compared with the previous decomposition strategy, this view decomposition is more granular,
```

Comment:
"previous" is ambiguous here because PV 6 was described **above**, but has been released **after** PV 5.
```suggestion
When compared with the decomposition strategy for protocol version 6, this view decomposition is more granular,
```